torch_geometric/datasets/fake.py

```diff
@@ -36,6 +36,7 @@ class FakeDataset(InMemoryDataset):
             an :obj:`torch_geometric.data.Data` object and returns a
             transformed version. The data object will be transformed before
             every access. (default: :obj:`None`)
+        **kwargs (optional): Additional attributes and their shapes *e.g.* :obj:`{'global_features':5}`.
```

Comment:
This is still a keyword arg, right? So `global_feature=5` should be correct.

Reply:
Yeah, I agree.